packages/slate-editor/src/components/ChooseGroup/ChooseGroup.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import type { ChooseGroupOption, CheckboxChoose, RadioChoose } from './ChooseOption';
+import { ChooseOption } from './ChooseOption';
+
+interface ChooseGroupCommon<T extends string> {
+    name: string;
+    options: ChooseGroupOption<T>[];
+    columns?: number;
+}
+
+type ChooseGroupProps<T extends string> = ChooseGroupCommon<T> &
+    (CheckboxChoose<T> | RadioChoose<T>);
+
+export function ChooseGroup<T extends string>(props: ChooseGroupProps<T>) {
```

Comment:
Hm... I don't remember ever seeing a component like this named `ChooseGroup`. Sounds awkward.

Can we check how other frameworks typically call these components?